test/unit/gateways/hps_test.rb

```diff
@@ -114,6 +124,14 @@ def test_successful_void
     assert_success void
   end
 
+  def test_successful_check_void
```

Comment:
I really wish any of the tests were actually checking the relevant XML for correctness! I may still do that here, since we'd like to make sure that we're sending check txns marked as such.

Reply:
Are you still working on getting the remote tests to work to verify that the XML is correct?

Reply:
I am; all the different tests do pass, but I have to use two different sets of creds to get them to do so. Since the provided creds won't have echeck support, it seems like it'll be especially useful.

Reply:
Resolving this because XML checks have been added AND the test creds have been managed!